src/intersection/compute_intersection.cc

```diff
@@ -742,10 +778,11 @@ unsigned int ComputeIntersection<Simplex<1>, Simplex<3>>::compute(IntersectionAu
     unsigned int count = compute(intersection.i_points_);
     
     // set if pathologic!!
+    /*
     for(IntersectionPointAux<1,3> &p : intersection.i_points_){
         if(p.is_pathologic()) intersection.pathologic_ = true;
         break;
-    }
+    }*/
     
```

Comment:
since tracing is not used, the flag "pathologic" is not needed at all in intersectionAux and IntersectionPointAux anymore